app/views/evaluations/index.html.erb

```diff
@@ -25,12 +27,19 @@
                   <td><%= user.level %></td>
                   <td><%= user.office_city%></td>
                   <td>
-                    <% unless user.evaluations.last.nil? %>
-                      <%= user.last_evaluation.evaluation_date || user.last_evaluation.created_at.strftime('%e/%m/%Y') %> -
-                      <%= link_to "View", evaluation_path(user.last_evaluation.id)%>
+                    <% unless user.last_performance_evaluation.nil? %>
+                      <%= user.last_performance_evaluation.evaluation_date || user.last_performance_evaluation.created_at.strftime('%e/%m/%Y') %> -
+                      <%= link_to "View", evaluation_path(user.last_performance_evaluation.id)%>
                     <% end%>
                   </td>
-                  <td><%= user.last_evaluation.score %></td>
+                  <td><%= user.last_performance_evaluation&.score %></td>
+                  <td>
+                    <% unless user.last_english_evaluation.nil? %>
+                      <%= user.last_english_evaluation.evaluation_date || user.last_english_evaluation.created_at.strftime('%e/%m/%Y') %> -
```

Comment:
A tip for a follow-up PR would be creating an Evaluation [viewmodel/presenter/decorator](https://www.nopio.com/blog/view-objects-rails-views/) to abstract complex logic like this:
```ruby
user.last_english_evaluation.evaluation_date || user.last_english_evaluation.created_at.strftime('%e/%m/%Y')
```

Also, we can leverage [Null Object Pattern](https://medium.com/@kelseydh/using-the-null-object-pattern-with-ruby-on-rails-b645ebf79785) there, so we can get rid of those `.try`s and `||`s, having less defensive programming.